[2024-03-11T14:00:09.855+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-06T00:00:00+00:00 [queued]>
[2024-03-11T14:00:10.048+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-06T00:00:00+00:00 [queued]>
[2024-03-11T14:00:10.087+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:10.231+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2021-08-06 00:00:00+00:00
[2024-03-11T14:00:10.277+0000] {standard_task_runner.py:60} INFO - Started process 2817 to run task
[2024-03-11T14:00:10.331+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'scheduled__2021-08-06T00:00:00+00:00', '--job-id', '711', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpdcfqujo8']
[2024-03-11T14:00:10.395+0000] {standard_task_runner.py:88} INFO - Job 711: Subtask second_task
[2024-03-11T14:00:10.990+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-06T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:12.099+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-06T00:00:00+00:00'
[2024-03-11T14:00:12.112+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:12.127+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and I ran after task1']
[2024-03-11T14:00:12.293+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:12.303+0000] {subprocess.py:93} INFO - hey, I am task2 and I ran after task1
[2024-03-11T14:00:12.330+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:12.503+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, execution_date=20210806T000000, start_date=20240311T140009, end_date=20240311T140012
[2024-03-11T14:00:12.894+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:13.694+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
